vectorvis/integrallinefiltering/src/processors/integrallinestodataframe.cpp

```diff
@@ -0,0 +1,383 @@
+                createFunction<T>(funcs, *df, percentiles, name + "", [c](const T &v) -> float {
+                    float l = 0;
+                    for (size_t i = 0; i < c; i++) {
+                        l += static_cast<float>(util::glmcomp(v, i) * util::glmcomp(v, i));
+                    }
+                    return std::sqrt(l);
+                });
+            }
+            if (x_.get()) {
+                createFunction<T>(funcs, *df, percentiles, name + "-x", [](const T &v) -> float {
+                    return static_cast<float>(util::glmcomp(v, 0));
+                });
+            }
+            if (y_.get()) {
+                createFunction<T>(funcs, *df, percentiles, name + "-y", [](const T &v) -> float {
+                    return static_cast<float>(util::glmcomp(v, 1));
+                });
+            }
+            if (z_.get()) {
+                createFunction<T>(funcs, *df, percentiles, name + "-w", [](const T &v) -> float {
+                    return static_cast<float>(util::glmcomp(v, 2));
+                });
+            }
+            if (w_.get()) {
+                createFunction<T>(funcs, *df, percentiles, name + "-z", [](const T &v) -> float {
+                    return static_cast<float>(util::glmcomp(v, 3));
+                });
+            }
+        });
+    }
+}
+
+const std::string IntegralLinesToDataFrame::MetaDataProperty::classIdentifier =
+    "org.inviwo.IntegralLinesToDataFrame.MetaDataProperty";
+
+// The Class Identifier has to be globally unique. Use a reverse DNS naming scheme
+const ProcessorInfo IntegralLinesToDataFrame::processorInfo_{
+    "org.inviwo.IntegralLinesToDataFrame",  // Class identifier
+    "Integral Lines To Data Frame",         // Display name
+    "Integral Line Filtering",              // Category
+    CodeState::Experimental,                // Code state
+    Tags::None,                             // Tags
+};
+const ProcessorInfo IntegralLinesToDataFrame::getProcessorInfo() const { return processorInfo_; }
+
+IntegralLinesToDataFrame::IntegralLinesToDataFrame()
+    : Processor()
+    , lines_("lines")
+    , dataframe_("dataframe")
+    , metaDataSettings_("metaDataSettings", "Meta Data Settings")
+    , includeLineID_("includeLineID", "Include Line ID", true)
+    , includeNumberOfPoints_("includeNumberOfPoints", "Include Number of points", false)
+
+    , includeLineLength_("includeLineLength", "Include Line Length", true)
+    , includeTortuosity_("includeTurtuosity", "Include Tortuosity", true)
+    , includeTerminationReason_("includeTerminationReason", "Include Termination Reasons", true)
+    , includeEntropy_("includeEntropy", "Include Entropy", true)
+    , includeStartPositions_("includeStartPositions", "Include Line Start Coordinates", false)
+    , includeEndPositions_("includeEndPositions", "Include Line End Coordinates", false) {
+
+    addPort(lines_);
+    addPort(dataframe_);
+
+    addProperty(includeLineID_);
```

Comment:
consider `addProperties`

Reply:
That function was not avaible when I implemented the processor ;)